### PR TITLE
FEATURE: Remove ignore feature SiteSetting and enable ignore by default

### PR DESCRIPTION
**Pull request by khalilovcmded**
## Why?

This is part of the [Ability to ignore a user feature](https://meta.discourse.org/t/ability-to-ignore-a-user/110254/8).

We are enabling the Ignore feature by default.

### CODE REVIEW

File: config/site_settings.yml [hunk 1]
```diff
@@ -529,9 +529,6 @@ users:
     default: false
     client: true
   log_personal_messages_views: false
-  ignore_user_enabled:
```
Comment (ZogStriP) [inline config/site_settings.yml]: We need a migration to remove that value form the database (`site_settings` table).

**Comment by khalilovcmded** [comment]
@ZogStriP oh, that's a good point! Just now I realised that these rows will always be persisted.

File: db/migrate/20190410102915_remove_ignore_user_enabled_site_setting.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class RemoveIgnoreUserEnabledSiteSetting < ActiveRecord::Migration[5.2]
```
Comment (khalilovcmded) [inline db/migrate/20190410102915_remove_ignore_user_enabled_site_setting.rb]: Will that work? @ZogStriP

Comment (ZogStriP) [reply db/migrate/20190410102915_remove_ignore_user_enabled_site_setting.rb]: Exactly 👍

**Pull request merged**